chore: add cli rules docs (#1819)
Redocly/redocly-cli: docs/rules/async/channels-kebab-case.md

### Channel rules for AsyncAPI 2.6

The syntax for how the channels are described changed with the AsyncAPI 3.0 release.

> If you still AsyncAPI 2.6, this rule works, and checks the channel address used as the key of the `channels` object.

Reviewer: Should be the same as in https://github.com/Redocly/redocly-cli/pull/1800
```suggestion
This rule also works with AsyncAPI 2.6 and checks the channel address used as the key of the `channels` object.
```
